assets/css/admwpp.css

```diff
@@ -1,3 +1,198 @@
+/* Colors */
+/* Font colors */
+/* Settings Page */
+/* Background colors */
+/* rows color */
+/* pager color */
+/* End Colors */
+/* Inputs */
+/* End inputs */
+/* Effects */
+/* End Effects */
+/* Fonts */
+/* Transition */
+/* Metaboxes Blocks */
+/* Colors */
+/* Fonts */
+/* Mixins */
+.selectric-wrapper {
```

Comment:
Selectric is a third party library so lets keep it outside the compiled CSS and enquey it as any other 3rd party library, and add the ovverides needed for it in a seperate sass file to be compiled with the rest of the files